.github/PULL_REQUEST_TEMPLATE/devconsole_pr.md

```diff
@@ -0,0 +1,29 @@
+
+# [JIRA-ID](add JIRA link): User Story / Defect / Task Title
+<!--For e.g [ODC-100](https://jira.coreos.com/browse/ODC-100): Title of user story / defect / task -->
```

Comment:
Not fond of the large `#` header size. Not sure if we need one here
Typically `fixes: https://jira.coreos.com/browse/ODC-###` is sufficient. I don't think the name of the issue is needed to be duplicated because it could get out of sync. The PR name is often good enough of a summary.

Reply:
Agree. Modified all sections title to h3.